x/concentrated-liquidity/tick.go

```diff
@@ -57,6 +57,9 @@ func (k Keeper) getTickInfo(ctx sdk.Context, poolId uint64, tickIndex int64) (ti
 	store := ctx.KVStore(k.storeKey)
 	tickStruct := model.TickInfo{}
 	key := types.KeyTick(poolId, tickIndex)
+	if !k.poolExists(ctx, poolId) {
+		return tickInfo, types.PoolNotFoundError{PoolId: poolId}
```

Comment:
nit: I know that these are equivalent. However, the change guarantees that we return an empty struct even if someone accidentally modifies the `tickInfo` prior to returning

```suggestion
		return model.TickInfo{}, types.PoolNotFoundError{PoolId: poolId}
```